phone-test/src/helpers/apolloClient.tsx

```diff
@@ -0,0 +1,50 @@
+import { ApolloClient, HttpLink, InMemoryCache, split } from "@apollo/client";
+import { setContext } from "@apollo/client/link/context";
+import { WebSocketLink } from "@apollo/client/link/ws";
+import { SubscriptionClient } from "subscriptions-transport-ws";
```

Comment:
[subscriptions-transport-ws](https://github.com/apollographql/subscriptions-transport-ws) is not maintained anymore, we need to switch to [graphql-ws](https://github.com/enisdenjo/graphql-ws) in the future (uses a different protocol)